### ISSUE TITLE
Support for decompress brotli-compressed sparse images

### ISSUE THREAD

**Issue by S-trace** [omnirom/android_packages_apps_OpenDelta#33]
Hello.

I'd like to use the OpenDelta updater on the LineageOS 17.1 ROMs, but with default build settings it gives very unsatisfiable results (686.3 MiB delta for 722.6 MiB target ROM), which makes an OpenDelta pointless.

But when I decompressed both system.dat.br and vendor.dat.br - opendelta.sh produced a delta file with just a 14.9 MiBs, which is appropriate for delta update main idea.

So, what about decompressing brotli-compressed sparse images before building a delta, and (maybe) recompressing images from resulting delta-updated final ZIP to save space on the device?

Thank you.

**Comment by darkobas** [comment]
Yes thats known. Disable brotli or choose what ever else works for you
